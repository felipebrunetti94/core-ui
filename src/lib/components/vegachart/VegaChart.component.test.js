import initStoryshots from "@storybook/addon-storyshots";

initStoryshots({
  configPath: "src/lib/components/vegachart/__snapshots__"
});
