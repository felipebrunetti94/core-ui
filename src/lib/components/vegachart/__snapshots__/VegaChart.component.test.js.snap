// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots VegaChart Default 1`] = `
<div>
  <h3>
    Vega-Lite wrapper - need to specify the entire spec
  </h3>
  <div
    className="sc-vegachart sc-bdVaJa bDWFJH"
    id="vis"
  />
</div>
`;
